maintainers/maintainer-list.nix

```diff
@@ -2615,6 +2615,12 @@
       fingerprint = "F549 3B7F 9372 5578 FDD3  D0B8 A1BC 8428 323E CFE8";
     }];
   };
+  br337 = {
+    name = "Brian Porumb";
```

Comment:
Sorry to be picky, but can you please sort attributes alphabetically? Just like most of other entries?

Reply:
Sure thing.

But keep in mind that the top comment in `maintainers-list.nix` isn't sorted in the correct order either. https://github.com/NixOS/nixpkgs/blob/4bfea513e208c4175fae8b59c0f94233b508c958/maintainers/maintainer-list.nix#L3-L16

The ordering should also be correct now

Reply:
This is something I might fix! Thanks for noticing it. That said, sorting attributes alphabetically is something common in Nix, so, the good habit is to sort them alphabetically.